crates/kitsune_p2p/kitsune_p2p/src/types.rs

```diff
@@ -109,15 +111,20 @@ macro_rules! make_kitsune_bin_type {
                 PartialOrd,
                 Ord,
                 shrinkwraprs::Shrinkwrap,
-                derive_more::From,
                 derive_more::Into,
                 serde::Serialize,
                 serde::Deserialize,
             )]
-            #[shrinkwrap(mutable)]
-            pub struct $name(#[serde(with = "serde_bytes")] pub Vec<u8>);
+            #[shrinkwrap(mutable, unsafe_ignore_visibility)]
```

Comment:
I can't find this `unsafe_ignore_visibility` in the shrinkwrap docs... is it so bad to have public data in newtypes?

Reply:
I wanted to ensure that there's only one path to construct this type, since we now have important debug assertions and some conditional initialization, so I feel it is bad to open up the possibility of a second path